src/ceph-volume/ceph_volume/activate/main.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from ceph_volume import terminal
+from ceph_volume.devices.lvm.activate import Activate as LVMActivate
+from ceph_volume.devices.raw.activate import Activate as RAWActivate
+from ceph_volume.devices.simple.activate import Activate as SimpleActivate
+
+
+class Activate(object):
+
+    help = "Activate an OSD"
+
+    def __init__(self, argv):
+        self.argv = argv
+
+    def main(self):
+        parser = argparse.ArgumentParser(
+            prog='ceph-volume activate',
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=self.help,
+        )
+        parser.add_argument(
+            '--osd-id',
+            help='OSD ID to activate'
+        )
+        parser.add_argument(
+            '--osd-uuid',
+            help='OSD UUID to active'
+        )
```

Comment:
I think we are missing the following argument in case we call `LVMActivate().activate()`

```py
        parser.add_argument(
            '--osd-fsid',
            help='OSD fs ID to activate'
        )
```

Reply:
the fsid vs uuid thing is annoying.  this gets passed to the lvm code as osd_uuid below.

Reply:
> the fsid vs uuid thing is annoying. this gets passed to the lvm code as osd_uuid below.

doh! yes... a bit annoying, more confusing than anything else